tests/test-data/fragment-html/sugar-img-test-fragment.html.out

```diff
@@ -22,12 +30,21 @@ Transformations made from HTML tags to AMP custom tags
 <img src="https://upload.wikimedia.org/wikipedia/commons/b/b7/True_Color_GIF_image-Tc217.gif"> at line 5
  ACTION TAKEN: img tag was converted to the amp-anim tag.
 
+<blockquote class="imgur-embed-pub" lang="en" data-id="UY7mHHT"> at line 8
+ ACTION TAKEN: blockquote.imgur imgur tag was converted to the amp-img tag
+
 
 AMP-HTML Validation Issues and Fixes
 -------------------------------------
-PASS
+FAIL
```

Comment:
An extra script tag in the test case was causing the amp validation to fail (which is expected). But I removed the script tag since it's not necessary for the unit test.